src/components/styles/calculator.css

```diff
@@ -1,9 +1,19 @@
+.calculate {
+  display: flex;
+
+  /* align-items: center; */
+  justify-content: space-between;
+  margin: 50px;
+}
+
 .calculator {
   display: flex;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  width: 100%;
+
+  /* align-items: center; */
+
+  /* justify-content: center; */
```

Comment:
- [ ] Same here, Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻